ROMFS/px4fmu_common/init.d/airframes/4041_beta75x

```diff
@@ -54,5 +54,34 @@ param set-default SYS_HAS_BARO 0
 param set-default SYS_HAS_MAG 0
 
 param set-default BAT1_N_CELLS 2
-# The Whoop uses reversed props
-set MIXER quad_h
+
+# Betaflight default motor ordering:
+#  4     2
+#     ^
+#  3     1
+param set-default SYS_CTRL_ALLOC 1
+param set-default CA_ROTOR_COUNT 4
+
+# rear right (CW)
+param set-default CA_ROTOR1_PX -0.707107
+param set-default CA_ROTOR1_PY 0.707107
+param set-default CA_ROTOR1_KM -0.05
+
+# front right (CCW)
+param set-default CA_ROTOR2_PX 0.707107
+param set-default CA_ROTOR2_PY 0.707107
+param set-default CA_ROTOR2_KM 0.05
+
+# rear left (CCW)
+param set-default CA_ROTOR3_PX -0.707107
+param set-default CA_ROTOR3_PY -0.707107
+param set-default CA_ROTOR3_KM 0.05
+
+# front left (CW)
+param set-default CA_ROTOR4_PX 0.707107
+param set-default CA_ROTOR4_PY -0.707107
+param set-default CA_ROTOR4_KM -0.05
+
+
```

Comment:
add default functions

Reply:
Is it on the main or aux?

Reply:
What do you mean by main vs aux? The BetaFPV quads afaik all use DSHOT only. I can test that airframe or was it tested before?